Documentation/ApiOverview/Fluid/_CustomViewHelper/_GravatarTagBasedViewHelper_getContentArgumentName.php

```diff
@@ -12,12 +12,17 @@ final class GravatarViewHelper extends AbstractTagBasedViewHelper
 
     public function initializeArguments(): void
     {
-        $this->registerArgument('emailAddress', 'string', 'The email address to resolve the gravatar for', false, null);
+        $this->registerArgument(
+            'emailAddress',
+            'string',
+            'The email address to resolve the gravatar for',
+            // The argument is optional now
+        );
     }
 
     public function render(): string
     {
-        $emailAddress = $this->arguments['emailAddress'] ?? $this->renderChildren();
+        $emailAddress = $this->renderChildren();
```

Comment:
As there is no string cast, that will result in error at `md5`:
```
Deprecated: md5(): Passing null to parameter #1 ($string) of type string is deprecated
```

Reply:
Could point, added some type checks